### ISSUE TITLE
Escaping characters with tmux.

### ISSUE THREAD

**Issue by jpambrun** [fbehrens/sublime_ExternalREPL#1]
Many characters need to be escaped when using tmux's send-keys command. For instance : 

```
   tmux send-keys -t repl "i=1" Enter
```

works fine while 

```
   tmux send-keys -t repl "i=1;" Enter
```

fails with "unknown command: Enter". On the other hand, 

```
   tmux send-keys -t repl "i=1\;" Enter
```

works flawlessly. 

The plugin is almost unusable with tmux because most statements end with semicolons. Other characters would also need to be escaped.

**Comment by adibender** [comment]
I can second @jpambrun's statement. The `$` sign is another example

**Comment by fbehrens** [comment]
Thanks, just found the issue now. Sorry. Will check

**Comment by fbehrens** [comment]
This is fixed in 0.1.3.
